docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/modernize-your-apps-lifecycle-with-ci-cd-pipelines-and-devops-tools-in-the-cloud.md

```diff
@@ -11,7 +11,7 @@ Although continuous integration and deployment practices are well established, t
 
 Azure DevOps Services supports continuous integration and deployment of multi-container applications to a variety of environments through the official Azure DevOps Services deployment tasks:
 
-- [Deploy to an Azure Web App for Containers](https://docs.microsoft.com/azure/devops/pipelines/apps/cd/deploy-docker-webapp?view=azure-devops)
+- [Deploy to an Azure Web App for Containers](https://docs.microsoft.com/azure/devops/pipelines/apps/cd/deploy-docker-webapp)
 
 - [Deploy to Azure Container Service – Kubernetes](https://docs.microsoft.com/azure/devops/build-release/apps/cd/azure/deploy-container-kubernetes)
```

Comment:
not related to your change but this link now goes to the Pipelines docs. @nishanil can you comment on what should be the appropriate change here? I believe this perhaps should go to https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/kubernetes/deploy?

Reply:
Added Suggestion.